source/basic.tex

```diff
@@ -4210,6 +4211,9 @@
 all the semantic restrictions shall be respected as if the temporary object
 had been created and later destroyed.
 \begin{note}
+A temporary object is unnamed.
```

Comment:
This new location is a bit unfortunate, because the next note talks about "This includes", which doesn't refer to unnamed temporaries, but to "semantic restrictions".

Reply:
I'm not moving down the note again, now it's after "This includes...".